assets/js/buttons-examples.js

```diff
@@ -804,7 +804,7 @@ $(document).ready(function () {
           totalWrittenClasses = `.${defaultClass} .${button.classes}`;
         if (thisButtonClasses.length) {
           totalClasses += ` ${thisButtonClasses}`;
-          totalWrittenClasses += ` .${thisButtonClasses}`;
+          totalWrittenClasses += ` .${thisButtonClasses.replace(/\s\s/g," ").replace(/\s/g , " .")}`;
```

Comment:
it can be just `.replace(" ", " .")` no need for two replace functions

Reply:
One thing we need to be careful is when classes are seperated by two or more spaces(note line 262, loading right red-btn), which is why I said using regex. Or we can just make it standard that all classes are seperated by one space.

Reply:
@shahednasser your suggestion doesnt work. as there are double spaces exist.

Reply:
@TomWBush thank you for pointing this out as I forgot to notice that. @Hanaffi then you can do the same solution but instead of the first parameter being `" "` it should be a regex. It still stands that there's no need for two `replace` calls

Reply:
@shahednasser I tried `.replace(/\s\s+/g, '  .')` 
but it didnt work.

Reply:
Your regex indicates that there should be at least two spaces. It should be `/\s+/g` instead.